pallets/ajuna-awesome-avatars/src/mock.rs

```diff
@@ -219,7 +195,7 @@ impl
 			initial_reward: Some(10),
 			max_reward: None,
 			take_fee_percentage: None,
-			reward_distribution: bounded_vec![40, 30, 10],
+			reward_distribution: vec![40, 30, 10].try_into().unwrap(),
```

Comment:
There was an unused import warning with some flags, this change removed the `bounded_vec` import altogether.